solutions/bio1/IntegerStrings/index.tsx

```diff
@@ -1,7 +1,60 @@
-import { STitle } from "components";
+import { Link, SCodeBlock, SText, STitle } from "components";
+import { brace } from "solutions/util";
 
 export const IntegerStrings = () => {
-  return <>
-    <STitle>Coming Soon!</STitle>
-  </>
-};
\ No newline at end of file
+  return (
+    <>
+      <STitle>Observation</STitle>
+      <SText>
+        This problem becomes much easier when you consider the string when $n=1$.
+      </SText>
+
+      <STitle>Sample Explanation</STitle>
+      <SText>
+        If we start at $999$, the string is $9991000100110021003...$. The $1$st digit is $9$, the $2$nd digit is also $9$, the $11$th digit is 1.
```

Comment:
seems to be some inconsistency here. please could you use dollar signs around all numbers please?

Reply:
also, I would prefer $1$-st and $2$-nd and so on, instead of $1$st and $2$nd.